### ISSUE TITLE
NEO field and chosen propagation method not saving to all entries when applied

### ISSUE THREAD

**Issue by AlexShearcroft** [AcclaroInc/craft-translations#301]
We're currently experiencing issues with our orders and them being applied to entries for multiple sites (languages).

When raising an order for an entry and selecting multiple sites (e.g. 5 sites) the NEO field blocks do not translate across all languages on "apply draft" or "merge and apply draft" but only one. This also happens if the same entry is listed in other orders and applied at a later time - it doesn't apply.

**Steps to reproduce**
- Raise an order for entry(s) with multiple target sites
- Export to XML and re-import updated XML content
- Select entries "ready for review" and then "review changes"
- Select all option and "merge to draft"
- Review the data - all changes in preview of XML and page look all fine to apply
- Apply the draft either via CMS entry or "Merge and apply draft"
- Check entries and only one target site is applied and all others are reset - even though in the XML preview this is still showing correctly

--

**Current versions and structural setups**
Craft Pro 3.7.30.1
Neo 2.12.3
Translations v2.1.2
 
**Section**
Propagation method: “Save entries to all sites enabled for this section”
 
**Fields**
Type: NEO
Propagation method: “Only save blocks to the site they were created in”
 
**NEO Block Types**
Field Type: Plain text
Translation method: “Translate for each language”
 
Field Type: Redactor
Translation method: “Translate for each language”

--

To summarise, all data for the NEO fields with the propagation method mentioned are showing in the correct fields upon XML upload, preview mode and within the entry via the CMS in the draft version for each language.
 
The issue is on applying/saving a draft, whereby data gets overwritten/reverted to original source for some reason.

**Comment by sidedwards** [comment]
@AlexShearcroft, this specific issue also happens with "localized" Neo blocks and native Craft drafts (i.e., outside the Translations plugin workflow). I opened an issue with [`spicywebau/craft-neo`](https://github.com/spicywebau/craft-neo), providing additional details and steps to reproduce.

Unfortunately, we were unable to find any additional workarounds aside from the propagation method update (switching from "Only save blocks to the site they were created in" to "Save blocks to all sites the owner element is saved in").

### 🔗 References
- https://github.com/spicywebau/craft-neo/issues/536
- https://github.com/AcclaroInc/craft-translations/discussions/249

**Comment by sidedwards** [comment]
@AlexShearcroft, this issue has been resolved in [Neo `2.12.4`](https://github.com/spicywebau/craft-neo/releases/tag/2.12.4).

The update also fixes a block duplication issue that often occurs in the same scenario. I would recommend manually removing any duplicated Neo blocks, updating Neo to the latest version, and retrying the translation workflow.

### 🔗 References
- https://github.com/spicywebau/craft-neo/issues/536#issuecomment-1026536947
- https://github.com/spicywebau/craft-neo/releases/tag/2.12.4
